### PR TITLE
Ckeditor and linter

### PR DESCRIPTION
**Pull request by jleveugle**

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build#522.0 ✘  
* build
  * build ✘  
  * test ✘

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (antleblanc) [inline .eslintrc]: indent_size = 2

File: package.json [hunk 2]
```diff
@@ -6,31 +6,49 @@
   "author": "OVH SAS",
   "repository": "ovh-ux/ovh-module-exchange",
   "scripts": {
-    "lint": "eslint src/. --ext .js --quiet",
-    "test": "grunt test"
+    "test": "echo 'TODO'",
+    "lint": "run-p lint:css lint:js",
+    "lint:css": "stylelint 'src/**/*.less' --fix",
+    "lint:js": "eslint --quiet --fix ./src",
+    "version": "npm-scripts-config version",
```
Comment (antleblanc) [inline package.json]: note: should be replaced but looks good for now.

File: package.json [hunk 3]
```diff
@@ -6,31 +6,49 @@
...
+    "postversion": "git push && git push --tags",
+    "preview-changelog": "npm-scripts-config preview-changelog"
   },
   "keywords": [
     "angular",
     "exchange"
   ],
+  "lint-staged": {
+    "*.less": [
```
Comment (antleblanc) [inline package.json]: note: missing "*.js"

File: .eslintrc (hunk 1 above)
Comment (antleblanc) [reply .eslintrc]: +`"root": true`

File: .eslintrc [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    "extends": "airbnb-base",
+    "parser": "babel-eslint",
+    "env": {
+        "browser": true,
+        "jquery": true,
+    },
+    "globals": {
+        "angular": true,
```
Comment (antleblanc) [inline .eslintrc]: Would be great to sort this list.

**Pull request merged**